From 9a5c900fdb532d553a78339a6887417dd6de70e5 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 16 Jan 2023 12:22:17 +0000
Subject: [PATCH] convert: windows: Document what copy_qemu_ga function returns

Just updates the comment.

(cherry picked from commit 7499ab10ea387b1dfd6fdd6feb2632b66545a257)
---
 convert/windows_virtio.mli | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/convert/windows_virtio.mli b/convert/windows_virtio.mli
index 73ec95c7..82aad69b 100644
--- a/convert/windows_virtio.mli
+++ b/convert/windows_virtio.mli
@@ -39,8 +39,11 @@ val install_drivers
     devices if we managed to install those, or legacy devices if we didn't. *)
 
 val copy_qemu_ga : Guestfs.guestfs -> Types.inspect -> string list
-(** copy MSIs (idealy just one) with QEMU Guest Agent to Windows guest. The
-    MSIs are not installed by this function. *)
+(** copy MSIs (ideally just one) with QEMU Guest Agent to Windows guest. The
+    MSIs are not installed by this function.
+
+    Returns a list of the copied [*.msi] files (empty list indicates no
+    qemu-ga installer(s) could be located). *)
 
 (**/**)
 
